tests/run-make/c-link-to-rust-va-list-fn/rmake.rs

```diff
@@ -3,12 +3,10 @@
 // prevent the creation of a functional binary.
 // See https://github.com/rust-lang/rust/pull/49878
 
-//@ ignore-cross-compile
-
-use run_make_support::{cc, extra_c_flags, run, rustc, static_lib_name};
+use run_make_support::{cc, extra_c_flags, run, rustc, static_lib_name, target};
 
 fn main() {
-    rustc().input("checkrust.rs").run();
+    rustc().input("checkrust.rs").target(target()).run();
```

Comment:
Remark: this will not be needed if I finish auto-cross #139244

Reply:
I removed `target(target()` bit now (yay!) and use the new `needs-target-std` in that test file.